perfkitbenchmarker/linux_benchmarks/netperf_benchmark.py

```diff
@@ -367,6 +371,16 @@ def RunNetperf(vm, benchmark_name, server_ip, num_streams):
   confidence = ('-I 99,5 -i {0},3'.format(FLAGS.netperf_max_iter)
                 if FLAGS.netperf_max_iter else '')
   verbosity = '-v2 ' if enable_latency_histograms else ''
+
+  netperf_cmd = ""
```

Comment:
Did you mean to set this to an empty string up here?  It is set on line 370/384

Reply:
That line has now been removed